.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: php
-matrix:
+jobs:
   include:
     - php: 7.1
     - php: 7.1
       env: deps=low
     - php: 7.2
     - php: 7.2
       env: deps=low
-
+    - php: 7.3
+    - php: 7.4
```

Comment:
There is a maintained fork which supports php 7.3, 7.4 and 8.0:

https://github.com/leanphp/phpspec-code-coverage/pull/35#issuecomment-537030001